### PR TITLE
fix: use new registry, correct smc brand

### PR DESCRIPTION
**Pull request by MariusRejdak**
I changed brand naming so it's in sync with backend

### CODE REVIEW

**Review by jarektomanek — APPROVED** [review]

File: src/bot.js [hunk 1]
```diff
@@ -120,7 +122,7 @@ class Bot {
     payload.components[componentName] =
       this.getFalconComponent(
         true,
-        `eu.gcr.io/deployment-pipeline/xc-r2d2-${skin}`,
+        `registry.k8s.xcaliber.io/platform/xcaf/${skin}`,
```
Comment (intellix) [inline src/bot.js:125]: doesn't need the tag?

**Review by amlynarski — APPROVED** [review]

**Review by picheli20 — APPROVED** [review]

File: src/bot.js (hunk 1 above)
Comment (MariusRejdak) [reply src/bot.js:125]: no, this registry is completely managed by us

Comment (intellix) [reply src/bot.js:125]: aight, thought the `:latest` issue was with the latest version of Docker Compose :D

**Review by intellix — APPROVED** [review]

File: src/bot.js (hunk 1 above)
Comment (MariusRejdak) [reply src/bot.js:125]: Ah, you mean this tag, it will be added by falcon webpanel using `branch` field

**Pull request merged**